releasenotes/notes/qiskit_2_deprecations-35cb57738ca1417e.yaml

```diff
@@ -0,0 +1,9 @@
+---
+deprecations:
+  - |
+    Remove the parts of the code that relied on deprecated `qiskit` components:
+      * Everything that uses `convert_to_target` is removed
+      * Usage of `qiskit.pulse` is removed.
+      * Use of `BackendProperties` is removed, resulting in the removal of :class:`.NoiseModel`'s :meth:`.from_backend_properties` method.
```

Comment:
`BackendProperties` is still mentioned in various docstrings (e.g. as a return value). This should be updated. It's also mentioned in _Device backend noise model simulations_ tutorial.

Reply:
This is actually far more serious than I thought. If we have no alternative to `BackendProperties` regarding the generation of noise models from real-device data we'll need to drop all of this code; however, I assume we have some solution in BackendV2.